src/components/new-request/search.vue

```diff
@@ -97,27 +122,8 @@
         </v-tooltip>
       </v-col>
 
-      <!-- Business Lookup/Fetch -->
-      <v-col v-if="showBusinessLookup" cols="12" md="6" class="business-lookup">
-        <template v-if="!business">
-          <BusinessLookup v-if="getIsAuthenticated" @business="onBusiness($event)"/>
-          <BusinessFetch v-else @business="onBusiness($event)"/>
-        </template>
-        <v-text-field
-          v-else
-          append-icon="mdi-close"
-          readonly
-          filled
-          hide-details
-          :label="business.identifier"
-          :value="business.legalName"
-          @click:append="onBusiness(null)"
-          @keyup.delete="onBusiness(null)"
-        />
-      </v-col>
-
-      <!-- once an entity type is selected (or Federal)... -->
-      <template v-if="entity_type_cd || isFederal">
+      <!-- once an entity type is selected, is Federal, or is Restorable -->
+      <template v-if="entity_type_cd || isFederal || isRestorable">
```

Comment:
The reason why I added `isRestorable` here is because when a user selects an entity that's not restorable (like LLP for example), we don't want to show this entire block.